index.html

```diff
@@ -1 +1,146 @@
+                    <li class="topNavItem">English</li>
+                    <li class="topNavItem">Mypage</li>
+                    <li class="topNavItem">Logout</li>
+                </ul> 
+            </nav>
+            <nav class="bottomNavBar">
+                <a href="index.html" target="_self" class="Logo"><img src="./Assets/NewLogo.png"></a>
+                <ul class="bottomNavList">
+                    
+                    <li class="bottomNavItem"><a href="about.html" target="_self" class="navLinks" >About</a></li>
+                    <li class="bottomNavItem"><a href="#Program" target="_self" class="navLinks" rel="noopener">Program</a></li>
+                    <li class="bottomNavItem"><a href="#speakers" target="_self" class="navLinks" rel="noopener">Speakers</a></li>
+                    <li class="bottomNavItem"><a href="#sponsor" target="_self" class="navLinks" rel="noopener">Sponsor</a></li>
+                    <li class="bottomNavItem"><a href="#" target="_self" class="navLinks" rel="noopener">News</a></li>
+                </ul>
+                <div class="voucher"><a href="#" target="_blank" rel="noopener">Catalogue</a></div>
+            </nav>
+            <nav class="mobileView">                
+                <i class="fas fa-bars" id="burgerIcon"></i>                
+                <div class="mobilemenu">
+                    <i class="fas fa-times"></i>
+                    <a class="mobile-menu-items home-page" href="index.html">Home</a>
+                    <a class="mobile-menu-items about-page" href="about.html">About</a>
+                    <a class="mobile-menu-items program-section" href="#Program">Program</a>
+                    <a class="mobile-menu-items speaker-section" href="#speakers">Speakers</a>
+                    <a class="mobile-menu-items sponsor-section" href="#sponsor">Sponsor</a>
+                    <a class="mobile-menu-items news-section" href="#">News</a> 
+                </div>
+            </nav>
 
+        </header>
+        <main>
+            <section class="homePage">
+                <div class="mainContent">
+                    <div class="headingContainer">
+                        <h2 class="greetingHeading">"Hello!Beauty  world"</h2>
+                        <h1 class="titleHeading">SUMMIT OF BEAUTY 2022</h1>
+                    </div>
+                    <div class="mainDescription">
+                        <p>Connect with the best experts, in real life, at our summit.
+                         Be part of this unique experience and listen to their
+                          life-changing advice in person!
+                        </p>
+                    </div>
+                    <div class="date-venue">
+                        <p class="date">2022.10.15(THU) ~ 16(FRI)</p>
+                        <p>@ National Museum of Dubai, Art Center Dubai and more</p>
+                    </div>
+                </div>
+            </section>
```

Comment:
**OPTIONAL** - There are some little variations in the text color for your page header. Please remember to stick to the general layout of the design template 👍 .

Your project(Please pay attention to the text color): 
<img width="627" alt="header text student" src="https://user-images.githubusercontent.com/73607512/197000782-c279d8c7-7f88-4ad6-8993-9f5225d0b879.PNG">

Design template:
<img width="624" alt="header text behnace" src="https://user-images.githubusercontent.com/73607512/197000858-dbf40d39-97f4-4453-9d22-7ad7bc8b48f6.PNG">